src/operator/pooling.cu

```diff
@@ -15,7 +15,17 @@ namespace op {
 template<>
 Operator *CreateOp<gpu>(PoolingParam param) {
 #if MXNET_USE_CUDNN == 1
-  return new CuDNNPoolingOp(param);
+  switch (param.pool_type) {
+    case pool_enum::kMaxPooling:
+      return new CuDNNPoolingOp(param);
+    case pool_enum::kAvgPooling:
+      return new CuDNNPoolingOp(param);
+    case pool_enum::kSumPooling:
+      return new PoolingOp<gpu, mshadow::red::sum>(param);
```

Comment:
Ah..It is changed. I also checked pooling.cc file. The type appears there too. Can you fix that one? I don't know how to add that one into this.